drivers/storage/portworx/components_test.go

```diff
@@ -10763,7 +10763,7 @@ func TestMonitoringMetricsEnabled(t *testing.T) {
 	require.Len(t, serviceMonitor.OwnerReferences, 1)
 	require.Equal(t, cluster.Name, serviceMonitor.OwnerReferences[0].Name)
 	require.Equal(t, expectedServiceMonitor.Spec, serviceMonitor.Spec)
-}
+}*/
```

Comment:
LOL -- guess that's one way to solve this.
* another way would be to copy updated `drivers/storage/portworx/testspec/prometheusRule.yaml` file from `px-rel-23.7.0` branch

Reply:
commented it for now

Reply:
That should be fine.. we won't carry this change into latest operator releases (23.7.0+) -- and we don't anticipate many more releases on this branch